samples/python/tutorial_code/videoio/video-input-psnr-ssim.py

```diff
@@ -86,8 +86,8 @@ def main():
 
     framenum = -1 # Frame counter
 
-    captRefrnc = cv.VideoCapture(sourceReference)
-    captUndTst = cv.VideoCapture(sourceCompareWith)
+    captRefrnc = cv.VideoCapture(cv.samples.findFile(sourceReference))
+    captUndTst = cv.VideoCapture(cv.samples.findFile(sourceCompareWith))
```

Comment:
Please use `findFileOrKeep` with `VideoCapture`.
Users may want to pass URI or GStreamer pipeline which are not handled by `findFile()` function.